src/attribute_type.rs

```diff
@@ -3787,6 +3804,36 @@ mod tests {
         assert!(parse_attribute("ssrc:foo").is_err());
     }
 
+    #[test]
+    fn test_anonymize_attribute_ssrc() -> Result<(), SdpParserInternalError> {
+        let mut anon = StatefulSdpAnonymizer::new();
+        let parsed =
+            parse_attribute("ssrc:2655508255 cname:{735484ea-4f6c-f74a-bd66-7425f8476c2e}")?;
+        let (ssrc1, masked) = if let SdpType::Attribute(a) = parsed {
+            let masked = a.masked_clone(&mut anon);
+            match (a, masked) {
+                (SdpAttribute::Ssrc(ssrc), SdpAttribute::Ssrc(masked)) => (ssrc, masked),
+                (_, _) => unreachable!(),
+            }
+        } else {
+            unreachable!()
+        };
+        assert_eq!(ssrc1.id, masked.id);
+        assert_eq!(ssrc1.attribute, masked.attribute);
+        assert_eq!("cname-00000001", masked.value.unwrap());
```

Comment:
Is it on purpose that you unwrap here, but then for ssrc2 further down you compare values directly instead (without unwrapping)?